script/test

```diff
@@ -71,7 +71,7 @@ for (let packageName in CONFIG.appMetadata.packageDependencies) {
     console.log(`Executing ${packageName} tests`.bold.green)
     const cp = childProcess.spawn(executablePath, testArguments)
     let stderrOutput = ''
-    cp.stderr.on('data', data => stderrOutput += data)
+    cp.stderr.on('data', data => (stderrOutput += data))
```

Comment:
Am I missing something here? Why not use curly brackets here? Surely the complaint is due to the assignment and return of the original statement?